amnestywebsite/humanity-theme
# Adds a 66/33 Chapter Block Pattern

Ref: https://github.com/amnestywebsite/humanity-theme/issues/172

Adds a 66/33 Chapter Block Pattern

**Steps to test**:
1. Edit a post
2. Open the block inserter, check the new pattern exists
3. Insert it and check it matches the example in the ticket

Example: http://bigbite.im/i/JCKZvC
